### PR TITLE
October dependency bumps 🌲 & action improvements

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    if: github.ref_name == 'dev' && needs.release-please.outputs.release_created
+
+    steps:
+      - name: 🚀 Deploy
+        uses: agrc/firebase-website-deploy-composite-action@v1
+        with:
+          identity-provider: ${{ secrets.IDENTITY_PROVIDER }}
+          service-account-email: ${{ secrets.SERVICE_ACCOUNT_EMAIL }}
+          project-id: ${{ secrets.PROJECT_ID }}
+          build-command: npm run build:stage
```
Comment (steveoh) [inline .github/workflows/push.yml:80]: **issue**: this needs the repo token

**Review by steveoh — CHANGES_REQUESTED** [review]
looks good!

File: .github/workflows/push.yml (hunk 1 above)
Comment (stdavis) [reply .github/workflows/push.yml:80]: Thanks for the catch!

Comment (stdavis) [reply .github/workflows/push.yml:80]: Actually, isn't that only required if preview == yes? Either way, I don't think that it's going to hurt.

**Review by steveoh — APPROVED** [review]

**Pull request merged**